### PR TITLE
Form fixes

### PR DESCRIPTION
**Pull request by Danielle-P**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -36,6 +36,9 @@ <h2>Northern Monk<br />Leeds</h2>
   </nav>
 
   <div class="content" id="rsvp-top">
+    <noscript id="javascript-disabled">
+      <p><span>&#9888 </span>Please enable Javascript to submit your RSVP</p>
```
Comment (benadamstyles) [inline src/index.html]:
```suggestion
      <p><span>&#9888</span>Please enable Javascript to submit your RSVP</p>
```

File: src/index.html [hunk 2]
```diff
@@ -45,55 +48,54 @@ <h2>RSVP</h2>
...
-      <form name="rsvp" method="POST" data-netlify="true">
+      <form id="rsvp" name="rsvp" method="POST" netlify-honeypot="bot-field" data-netlify="true">
+        <p class="bot-field">
+          <label>Don’t fill this out if you're human: <input name="bot-field" /></label>
+        </p>
 
         <div class="guest-fields active">
-          <label>Guest 1<input type="text" name="name" placeholder="Full Name" required /></label>
-          <label class="checkbox-label"><input type="checkbox" name="vegetarian" value="vegetarian" />Vegetarian</label>
+          <label>Guest 1<input type="text" name="First Guest Name" placeholder="Full Name" required /></label>
```
Comment (benadamstyles) [inline src/index.html]: `Guest 1 Name` etc

File: src/index.js [hunk 3]
```diff
@@ -24,13 +25,34 @@ if (guestSelect) {
...
       } else {
         guestField.classList.remove('active')
+        guestField.querySelector(guestNameSelector).removeAttribute('required')
       }
     })
   })
 }
+
+rsvpForm.addEventListener('change', function(event) {
+  var dietaryRequirementSelected = event.target
```
Comment (benadamstyles) [inline src/index.js]: rename variable to `changedElement`

**Review by benadamstyles — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (Danielle-P) [reply src/index.html]: Done

**Review by benadamstyles — APPROVED** [review]

**Pull request merged**